holopy/core/io/io.py

```diff
@@ -380,6 +380,96 @@ def save_image(filename, im, scaling='auto', depth=8):
         pilimage.fromarray(im).save(filename)
 
 
+def save_images(filenames, ims, scaling='auto', depth=8):
+    """
+    Saves a volume as separate images (think of reconstruction volumes).
+
+    Parameters
+    ----------
+    ims : ndarray or :class:`holopy.image.Image`
+        Images to save, with separate z-coordinates from which they each will
+        be selected.
+    filename : str
+        Filename in which to save image. Has to contain a format placeholder
+        for the z-values. E.g. "testfile%f.png"
+    scaling : 'auto', None, or (Int, Int)
+        How the images should be scaled for saving. Ignored for float output.
+        It defaults to auto, use the full range of the output format. Other
+        options are None, meaning no scaling, or a pair of integers specifying
+        the values which should be set to the maximum and minimum values of the
+        image format.
+    depth : 8, 16 or 'float'
+        What type of image to save. Options other than 8bit may not be
+        supported for many image types. You probably don't want to save 8bit
+        images without some kind of scaling.
```

Comment:
If you can think of a pithy minimal-working example here for the docs, that would be awesome. If not, no worries.

Reply:
An example for the depth-Parameter? I'm not sure – as I just copied it other. Or of the whole method itself?

Reply:
Sorry! I meant an example of the whole method itself.

Reply:
Okay, no problem. In the actual docs or inside of this comment?